### DISCUSSION TITLE
如何部署github action里面的workflow

### DISCUSSION THREAD

**Discussion by moringspeaker** [somenothing/SCU-ncov_checkpoint#8] [Q&A]
我自己将写好的github workflow 设置为自动运行，但是却发现每天运行的是之前版本的workflow文件，而不是最新的，我感觉很疑惑，这是我的项目地址：https://github.com/moringspeaker/SCU_Wechat_Sign_in

**Comment by Fuzzy9876** [comment]
workflow我也没用过，不过还是推荐用服务器或者个人电脑运行。另外不建议直接在仓库里放出secret key和一切有可能导致泄露的隐私信息（比如真实的地理位置返回值）。
